### PR TITLE
Draft a CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by workingjubilee**
Just an initial pass, please feel free to leave any remarks/requests you have.

### CODE REVIEW

**Review by calebzulawski — APPROVED** [review]

**Comment by workingjubilee** [comment]
Was not entirely satisfied with the first version so I took another pass over it, it especially occurred to me that stating where a lot of other SIMD-relevant details are located may be pertinent.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
 Simple version:
-1. Fork it and git clone it
-2. Create your feature branch: git checkout -b my-branch
+1. Fork it and `git clone` it
+2. Create your feature branch: `git checkout -b my-branch`
 3. Write your changes.
-4. Test it: cargo test. Remember to use appropriate features.
-5. Commit your changes: git commit add ./path/to/changes && git commit -m 'Fix some bug'
-6. Push to the branch: git push origin my-branch
+4. Test it: `cargo test --features `. Remember to use appropriate SIMD features.
```
Comment (calebzulawski) [inline CONTRIBUTING.md:7]: Do we think we'll ever have any cargo features?

Comment (workingjubilee) [reply CONTRIBUTING.md:7]: ...No, that was an error!

Comment (workingjubilee) [reply CONTRIBUTING.md]: I had meant to say "set RUSTFLAGS", which it now does.

**Pull request merged**